yb-voyager/cmd/archiveChangesCommand.go

```diff
@@ -0,0 +1,243 @@
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+var archiveChangesCmd = &cobra.Command{
+	Use:   "changes",
+	Short: "This command will archive the streaming data from the source database",
+	Long:  `This command will archive the streaming data from the source database.`,
+
+	PreRun: func(cmd *cobra.Command, args []string) {
+		validateCommonArchiveFlags()
+	},
+
+	Run: archiveChangesCommandFn,
+}
+
+func archiveChangesCommandFn(cmd *cobra.Command, args []string) {
+	var err error
+	metaDB, err = NewMetaDB(exportDir)
+	if err != nil {
+		utils.ErrExit("Failed to initialize meta db: %s", err)
+	}
+
+	moveToChanged := cmd.Flags().Changed("move-to")
+	deleteChanged := cmd.Flags().Changed("delete")
+	if moveToChanged == deleteChanged {
+		utils.ErrExit("Either move-to or delete flag should be specified")
+	}
+
+	copier := NewEventSegmentCopier(moveDestination)
+	deleter := NewEventSegmentDeleter(utilizationThreshold)
+	var wg sync.WaitGroup
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		err := copier.Run()
+		if err != nil {
+			utils.ErrExit("copying segments: %v", err)
+		}
+	}()
+
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		err := deleter.Run()
+		if err != nil {
+			utils.ErrExit("deleting segments: %v", err)
+		}
+	}()
+
+	wg.Wait()
+}
+
+func init() {
+	archiveCmd.AddCommand(archiveChangesCmd)
+	registerCommonArchiveFlags(archiveChangesCmd)
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////
+
+type EventSegmentDeleter struct {
+	FSUtilisationThreshold int
+}
+
+func NewEventSegmentDeleter(fsUtilisationThreshold int) *EventSegmentDeleter {
+	return &EventSegmentDeleter{
+		FSUtilisationThreshold: fsUtilisationThreshold,
+	}
+}
+
+func (d *EventSegmentDeleter) isFSUtilisationExceeded() bool {
+	fsUtilization, err := utils.GetFSUtilizationPercentage(exportDir)
+	if err != nil {
+		utils.ErrExit("get fs utilization: %v", err)
+	}
+
+	return fsUtilization > d.FSUtilisationThreshold
+}
+
+func (d *EventSegmentDeleter) deleteSegment(segment Segment) error {
+	if utils.FileOrFolderExists(segment.FilePath) {
+		err := os.Remove(segment.FilePath)
+		if err != nil {
+			return fmt.Errorf("delete segment file %s: %v", segment.FilePath, err)
+		}
+	}
+	err := metaDB.MarkSegmentDeleted(segment.Num)
+	if err != nil {
+		return fmt.Errorf("mark segment %d as deleted: %v", segment.Num, err)
+	}
+	utils.PrintAndLog(fmt.Sprintf("event queue segment file %s deleted", segment.FilePath))
```

Comment:
`utils.PrintAndLog("event queue segment file %s deleted", segment.FilePath)`
Check the signature of the `utils.PrintAndLog()` function.

Reply:
Fixed